content/en/getting-started/quickstart/index.md

```diff
@@ -156,45 +163,45 @@ You can use the following command to subscribe an email address to the SNS topic
 
 {{< command >}}
 $ awslocal sns subscribe \
-    --topic-arn arn:aws:sns:us-east-1:000000000000:failed-resize-topic \
-    --protocol email \
-    --notification-endpoint my-email@example.com
+ --topic-arn arn:aws:sns:us-east-1:000000000000:failed-resize-topic \
+ --protocol email \
+ --notification-endpoint my-email@example.com
 {{< / command >}}
 
 #### Create the Presign Lambda
 
 {{< command >}}
 $ (cd lambdas/presign; rm -f lambda.zip; zip lambda.zip handler.py)
 $ awslocal lambda create-function \
-    --function-name presign \
-    --runtime python3.9 \
-    --timeout 10 \
-    --zip-file fileb://lambdas/presign/lambda.zip \
-    --handler handler.handler \
-    --role arn:aws:iam::000000000000:role/lambda-role \
-    --environment Variables="{STAGE=local}"
+ --function-name presign \
+ --runtime python3.9 \
```

Comment:
minor suggestion: Would it make sense to upgrade to python3.12 because 3.9 might get deprecated next year (it's the latest non-deprecation Python version) https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Thank you @tinyg210 for tackling the missing build instructions, that's indeed a common issue 👍